### PR TITLE
Warning that a newer version of core template is available

### PR DESCRIPTION
**Pull request by i8abyte**
[ADO Card: USER STORY 5522](https://dev.azure.com/S72/SHIFT72/_workitems/edit/5522)
[AC Doc](https://docs.google.com/document/d/1HPesMS_zpZhHUmB19jseHg1nngYH7mwQgH_TJlcYcxI/edit?pli=1#heading=h.w3jg0f6sfg5c)

### CODE REVIEW

**Review by smitsanghvi21 — APPROVED** [review]
Works fine when tested locally. Would be good to wait for @stajs or @PeculiarGoat review before merging.

**Review by PeculiarGoat — CHANGES_REQUESTED** [review]
Some things to change.

Also, I'm not sure if there is a need for all the date conversions (number to string and back, etc). We can have a dev session if you need a hand.

**Comment by stajs** [comment]
@brynyShift72 I think you are going to take over this one? I'll close this PR and you can submit your own when ready.

**Pull request closed without merging**
